source/includes/table-beta-compatibility.rst

```diff
@@ -50,11 +56,29 @@
      - |checkmark|
      - 
      - |checkmark|
+     - |checkmark|
 
    * - Destination Data Handling 
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
-     - 
\ No newline at end of file
+     - 
+     - |checkmark|
+
+   * - Pre-6.0 Version Support
+     - |checkmark|
+     - |checkmark| [#pre-6.0-abc]_
+     - 
+     -
+     - |checkmark|
+     - |checkmark|
+     -
+
+.. [#pre-6.0-abc]
+
+   A->B->C migrations are compatible with pre-6.0 version support only if 
```

Comment:
for some reason the staging link includes two footnotes (1, 2) and clicking the "2" link leads to an empty space at the bottom of the page

Reply:
thank you!